### DISCUSSION TITLE
Add state_class to Entity

### DISCUSSION THREAD

**Discussion by slinkingant** [zachowj/node-red-contrib-home-assistant-websocket#459] [Feature Request]
Can you add a 'state_class' and 'last_reset' to the optional configuration of an Entity-sensor?

I've created a flow to fetch and parse the generated power from my Solar inverter, but I can't select it to use in the new Energy dashboard due to the right state_class not being provided by the sensor.

See:
https://www.home-assistant.io/more-info/statistics/
https://developers.home-assistant.io/docs/core/entity/sensor/#long-term-statistics

**Comment by zachowj** [comment]
Yes, these need to be added. In the interim, I believe you can set these values as attributes of the entity and they should be picked up.

**Comment by zachowj** [comment]
Added in v0.40.0
